aws-xray-auto-instrumentation-agent-bootstrap/src/main/java/com/amazonaws/xray/agent/XRayInstrumentationAgent.java

```diff
@@ -111,13 +111,12 @@ private static AgentRuntimeLoaderInterface getAgentRuntimeLoader(ClassLoader cla
      * NOTE: THIS WILL BE REMOVED WHEN THE AGENT COMES OUT OF BETA
      * Parses the name for segments from JVM command line args. Going forward this name will be taken from
      * a configuration file.
-     * @param agentArgs
-     * @param defaultName
-     * @return
+     * @param agentArgs the string
+     * @return the service name parsed from command line args, or null if none was discovered
      */
-    private static String getServiceNameFromArgs(String agentArgs, String defaultName) {
+    private static String getServiceNameFromArgs(@Nullable String agentArgs) {
```

Comment:
`@Nullable` on return type too

```suggestion
    @Nullable
    private static String getServiceNameFromArgs(@Nullable String agentArgs) {
```

Reply:
Ok it goes on both, makes sense.